.github/workflows/release.yml

```diff
@@ -329,3 +328,30 @@ jobs:
     secrets: inherit
     with:
       ziti-version: ${{ needs.publish.outputs.ZITI_VERSION }}
+
+  repository-dispatch:
+    if: ${{ !cancelled() && needs.publish.result == 'success' }}
+    needs:
+      - publish
+      - call-publish-docker-images
+    name: Trigger Repository Dispatch
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Git Checkout
```

Comment:
is the checkout required?

Reply:
I remembered incorrectly. The checkout isn't required for `gh api` because it's a repo-less operation, no `--repo` or Git checkout introspection.